assets/javascripts/discourse/connectors/topic-footer-main-buttons-before-create/assign-button.js.es6

```diff
@@ -3,7 +3,7 @@ import { ajax } from 'discourse/lib/ajax';
 
 export default {
   shouldRender(args, component) {
-    const needsButton = component.currentUser && component.currentUser.get('staff');
+    const needsButton = component.currentUser && component.currentUser.get('staff') && component.siteSettings.assign_enabled;
```

Comment:
It is inside the `disabled_plugins`, but the button on topic end still appears:

![image](https://cloud.githubusercontent.com/assets/1385470/26656131/30fe8560-4634-11e7-94b7-95e9d8764227.png)